components/tools/OmeroPy/src/omero/plugins/admin.py

```diff
@@ -1063,6 +1069,7 @@ def copy_template(input_file, output_dir):
 
         return rv
 
+    @windows_warning
```

Comment:
Conceivably, `with_config` could be updated to include `windows_warning` if that would be cleaner. But this makes sense too.